src/Components/ModalCard/styles.ts

```diff
@@ -0,0 +1,77 @@
+import { Card, CardActionArea, styled as muiStyled } from "@mui/material";
+import { styled } from "styled-components";
+import { colors } from "../../global-styles";
+
+export const StyledCard = muiStyled(Card)`
+  width: 502px;
+  height: 569px;
+`;
+
+export const StyledCardActionArea = muiStyled(CardActionArea)`
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  padding: 24px;
+  align-items: center; // Centers all content horizontally
+  gap: 10px;
+`;
+
+export const StyledImg = styled.img`
+  width: 190px;
+  height: 150px;
+`;
+
+export const ImgContainer = styled.div`
+  width: 454px;
+  height: 158px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`;
+
+export const HeaderContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center; // Centers header content vertically
+  align-items: center; // Centers header content horizontally
+
+  & > * + * {
```

Comment:
what is this selector?

Reply:
it is for applying styles to every child element , in my case is for a top space of 10px.